### PR TITLE
Checking for port-SG connection

### PR DESCRIPTION
**Pull request by szpotona**
Good enough fix for race condition that is checking in OpenStack API if security group is in port security groups list. If not, update port is called again. Race condition is less probable but still possible.

### CODE REVIEW

File: neutron_plugin/port.py [hunk 1]
```diff
@@ -45,6 +45,8 @@
 RUNTIME_PROPERTIES_KEYS = \
     COMMON_RUNTIME_PROPERTIES_KEYS + [FIXED_IP_ADDRESS_PROPERTY]
 
+_NO_SG_PORT_CONNECTION_RETRY_INTERVAL = 3
```
Comment (Jessiebr) [inline neutron_plugin/port.py]: please change to NO_SG_PORT_CONNECTION_RETRY_INTERVAL (consistency)

File: neutron_plugin/port.py [hunk 2]
```diff
@@ -166,6 +168,18 @@ def connect_security_group(neutron_client, **kwargs):
     sgs = port['security_groups'] + [security_group_id]
     neutron_client.update_port(port_id, {'port': {'security_groups': sgs}})
 
+    # Double check if SG has been actually updated (a race-condition
+    # in OpenStack):
+    port_info = neutron_client.show_port(port_id)['port']
+    port_security_groups = port_info.get('security_groups', [])
+    if security_group_id not in port_security_groups:
```
Comment (Jessiebr) [inline neutron_plugin/port.py]: please verify that the tests cover this

**Comment by Jessiebr** [comment]
Please squash

File: neutron_plugin/port.py (hunk 2 above)
Comment (szpotona) [inline neutron_plugin/port.py]: Updated. Now tests cover this.

**Pull request merged**